src/server/routes/generators/typescript.ts

```diff
@@ -0,0 +1,200 @@
+import { FastifyInstance } from 'fastify'
+import prettier from 'prettier'
+import { PostgresMeta, PostgresType } from '../../../lib'
+import { DEFAULT_POOL_CONFIG } from '../../constants'
+import { extractRequestForLogging } from '../../utils'
+
+export default async (fastify: FastifyInstance) => {
+  fastify.get<{
+    Headers: { pg: string }
+    Querystring: {
+      excluded_schemas?: string
```

Comment:
this is for `auth`, `storage`, `graphql`, `realtime` etc. - we should only generate for user's schemas